### PR TITLE
Update shelley-genesis.md

### LINKED ISSUE #1613: [BUG] - Shelley genesis walkthrough docs, outdated?
**Internal/Exernal**
*External*   otherwise.

**Summary**
Walkthrough steps do not seem to be correct.  https://docs.cardano.org/projects/cardano-node/en/latest/reference/shelley-genesis.html

**Steps to reproduce**

Following with all steps we end up with a `example` directory with contents:
```
example
├── delegate-keys
│  ├── delegate-opcert1.counter
│  ├── delegate-opcert2.counter
│  ├── delegate1.skey
│  ├── delegate1.vkey
│  ├── delegate2.skey
│  └── delegate2.vkey
├── genesis-keys
│  ├── genesis1.skey
│  ├── genesis1.vkey
│  ├── genesis2.skey
│  └── genesis2.vkey
├── genesis.json
├── genesis.spec.json
└── utxo-keys
   ├── utxo1.skey
   ├── utxo1.vkey
   ├── utxo2.skey
   └── utxo2.vkey
``` 

```
cardano-cli shelley genesis create --genesis-dir example/ --mainnet 
Mismatch between the files found:
Genesis key file indexes:      [1,2]delegate key file indexes:     [1,2]delegate VRF key file indexes: []
```

This seems to be missing the VRF keys, which is documented further down. However to generate those it seems we need to generate something `node1` related (I assume node keys):
```
cardano-cli shelley node key-gen-VRF \
    --verification-key-file example/node1/vrf.vkey \
    --signing-key-file example/node1/vrf.skey
example/node1/vrf.skey: example/node1/vrf.skey: openBinaryFile: does not exist (No such file or directory)
```

However... That section has:
> For now we will ignore stake pools however since we need to get the system bootstrapped with the BFT overlay.

Which makes me think I might be going down a rabbit hole, so possibly these instructions are outdated or incorrect?

**Expected behavior**

Being able to successfully update the genesis files.

---

Also I could mention that all the commands upto now are seem to be missing a `--mainnet` parameter (though this was easy to figure out and just add in).

### PR DESCRIPTION
**Pull request by Jimbo4350**
Fixes #1613

### CODE REVIEW

File: doc/reference/shelley-genesis.md [hunk 1]
```diff
@@ -120,32 +124,37 @@ $ cardano-cli shelley genesis key-gen-genesis \
...
- 582066361ff2e1b5fdf25bc14bc48424b64fa62cee7692ed7e86216eafd3bdb28cbd
+{
+    "type": "GenesisVerificationKey_ed25519",
+    "description": "Genesis Verification Key",
+    "cborHex": "5820562ede753089653ee876b7f97c6f93435a320f6581423b73441ad24838c940fd"
+}
 ```
 
+
 The "type" must not be edited. This is used as a sanity check. The "title"
```
Comment (intricate) [inline doc/reference/shelley-genesis.md:138]:
```suggestion
The "type" must not be edited. This is used as a sanity check. The "description"
```

File: doc/reference/shelley-genesis.md [hunk 2]
```diff
@@ -1295,26 +1394,47 @@ We will be using the low level `build-raw` command because a more convenient
...
+  --json-metadata-no-schema
+                           Use the "no schema" conversion from JSON to tx
+                           metadata.
+  --json-metadata-detailed-schema
+                           Use the "detailed schema" conversion from JSON to tx
+                           metadata.
+  --metadata-json-file FILE
+                           Filepath of the metadata file, in JSON format.
+  --metadata-cbor-file FILE
+                           Filepath of the metadata, in raw CBOR format.
```
Comment (newhoggy) [inline doc/reference/shelley-genesis.md:1434]: Should this also say "metadata file"?

**Review by newhoggy — APPROVED** [review]

File: doc/reference/shelley-genesis.md (hunk 2 above)
Comment (intricate) [reply doc/reference/shelley-genesis.md:1434]: That description for `metadata-json-file` should probably be changed, actually. "Filepath of the metadata file" sounds redundant.

**Review by intricate — APPROVED** [review]

**Pull request merged**